packages/editor/src/components/rich-text/index.js

```diff
@@ -169,13 +169,14 @@ export class RichText extends Component {
 		} );
 	}
 
-	applyRecord( record ) {
+	applyRecord( record, { domOnly } = {} ) {
 		apply( {
 			value: record,
 			current: this.editableRef,
 			multilineTag: this.multilineTag,
 			multilineWrapperTags: this.multilineWrapperTags,
 			prepareEditableTree: this.props.prepareEditableTree,
+			__unstableDomOnly: domOnly,
```

Comment:
Part of me wonders if this is the sort of thing where it'd be easier to just pass through whatever additional options verbatim, i.e. `...options` from a second object argument. But I can also see the opposite argument of there being value in being explicit, especially in this case where it's something perhaps temporary.

Reply:
Yeah, I definitely see this as something temporary.